File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.cs
public class Solution {
    public int[] ReplaceElements(int[] arr) {
        int[] result = new int[arr.Length];
        for(int i = 0 ; i < arr.Length - 1; i++)
        {
           int maxRight = arr[i+1]; 
           for(int j = i+1 ; j < arr.Length ; j++)
           {
            if(maxRight < arr [j]){
                maxRight = arr[j];
            }
           }
           result[i] = maxRight;
        }
        result[arr.Length - 1] = -1;
        return result;
    }
}

File: Data Structures & Algorithms/count-the-number-of-consistent-strings/submission-0.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        hash_set = set(allowed) # We can initialize the set directly like this!
        res = 0

        for word in words:
            is_consistent = True # Reset the flag for each new word
            
            for char in word:
                if char not in hash_set:
                    is_consistent = False # Flag it as inconsistent
                    break                 # Stop checking this word
            
            # Only add to our result if the flag never changed to False
            if is_consistent:
                res += 1

        return res